ams/ams-server/src/main/java/com/netease/arctic/ams/server/ArcticMetaStore.java

```diff
@@ -204,6 +209,13 @@ public static boolean isStarted() {
   public static void failover() {
     stopMetaStore();
     AmsRestServer.stopRestServer();
+    isLeader.set(false);
+  }
+
+  public static void shutDown() {
```

Comment:
please change `com.netease.arctic.ams.server.AmsEnvironment#stopAms` to use `ArcticMetaStore.shutDown()` in this PR.

Reply:
will change in new commit